ui/src/components/NodePartitionTable.test.js

```diff
@@ -52,7 +53,12 @@ const server = setupServer(
               'https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-nodefilesystemalmostoutofspace',
             summary: 'Filesystem has less than 5% space left.',
           },
-          endsAt: '2021-01-29T07:40:05.358Z',
+          /*
+          We want to have an active alert, meaning the current time should be in between `startsAt` and `endsAt`.
+          Since we can't spyon the `activeOn` in getHealthStatus(). If we use ```endsAt: new Date().toISOString()```, there will be slightly bwtween the two current time.
```

Comment:
```suggestion
          Since we can't spy on the `activeOn` in getHealthStatus(). If we use ```endsAt: new Date().toISOString()```, there will be a slightly difference between the two current time.
```